src/status_im/ui/screens/profile/styles.cljs

```diff
@@ -153,22 +154,25 @@
    :android            {:height 72}
    :ios                {:height 64}})
 
-(def network-settings-text
-  (merge {:flex 1}
-         profile-setting-text))
+(defstyle settings-item-text
+  {:flex      1
+   :font-size 15
```

Comment:
It would be great to have global vars for those numbers

Reply:
That's the same point as with colors - font sizes are something that can be stanardized as well.

Reply:
would be cool, with comments